src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const newCalculator = {
+
+    result: 0,
+
+    add(value) {
+      newCalculator.result += value;
+    },
+
+    subtract(value) {
+      newCalculator.result -= value;
+    },
+
+    multiply(value) {
+      newCalculator.result *= value;
+    },
+
+    divide(value) {
+      newCalculator.result /= value;
+    },
+
+    reset() {
+      this.result = 0;
```

Comment:
why do you use `newCalculator.result` in all others methods, but `this.result` here?